tools/redpanda-connect/generate-rpcn-connector-docs.js

```
@@ -0,0 +1,202 @@
+ */
+function mergeOverrides(target, overrides) {
+  if (!overrides || typeof overrides !== 'object') return target;
+  for (const key in overrides) {
+    if (Array.isArray(target[key]) && Array.isArray(overrides[key])) {
+      // Merge two parallel arrays by matching items on `.name`
+      target[key] = target[key].map(item => {
+        const overrideItem = overrides[key].find(o => o.name === item.name);
+        if (overrideItem) {
+          // Overwrite description/type if present
+          ['description', 'type'].forEach(field => {
+            if (overrideItem.hasOwnProperty(field)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)